Classes/Domain/Service/FakeNodeDataFactory.php

```diff
@@ -84,7 +84,7 @@ public function createFromPayload(array $payload): NodeData
         $nodeData->setRemoved(true);
 
         // Ensure, the fake-node is not persisted
-        if ($this->persistenceManager->isNewObject($nodeData)) {
+        if (false === $this->persistenceManager->isNewObject($nodeData)) {
```

Comment:
```suggestion
        if ($this->persistenceManager->isNewObject($nodeData) === false) {
```
we normally don't do Yoda-Style